webapp/src/components/common/Matrix/components/MatrixActions/index.tsx

```diff
@@ -73,18 +66,25 @@ function MatrixActions({
           </IconButton>
```

Comment:
Disable undo and redo buttons when submitting.

Reply:
thanks